src/core/components/table/AttendanceTable.vue

```diff
@@ -213,6 +210,12 @@ export default {
     traineesWithAttendance () {
       return [...this.course.trainees, ...this.unsubscribedTrainees];
     },
+    formattedAttendanceSheet () {
+      return this.attendanceSheets.map(as => ({
```

Comment:
et il ne faut faire cette manip que pour les formations INTER_B2B
dasn le cas d'une INTRA il faut que tu retournes simplement `this.attendanceSheet`
je pense que l'erreur que j'ai viens de la

Reply:
done